### PR TITLE
Support profiling E2E

### PR DESCRIPTION
**Pull request by mrproliu**

### CODE REVIEW

File: .asf.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ github:
       required_status_checks:
         strict: true
         contexts:
-          - Build and Test
+          - Required
```
Comment (kezhenxu94) [inline .asf.yaml:38]: You can simply remove the original one, that prevents your PR being merged

Comment (mrproliu) [reply .asf.yaml:38]: I have removed the matrix in the `Build and Test` job.

Comment (kezhenxu94) [reply .asf.yaml:38]: You still need to restore the original one `Build and Test` back

File: .github/workflows/rover.yaml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        if: matrix.runner == 'ubuntu'
+        uses: apache/skywalking-eyes@9bd5feb86b5817aa6072b008f9866a2c3bbc8587
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Get dependencies
+        run: make deps
+      - name: Generate
+        run: make container-generate
+      - name: Test
+        run: make container-test
```
Comment (kezhenxu94) [inline .github/workflows/rover.yaml:46]: `test` goal already depends on `generate` so we don't need to split the `generate` goal individually, which might save some time for CI

**Review by kezhenxu94 — COMMENTED** [review]
Others LGTM

File: .github/workflows/rover.yaml (hunk 2 above)
Comment (mrproliu) [reply .github/workflows/rover.yaml]: Updated. Thanks.

File: .github/workflows/rover.yaml [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+          go-version: 1.17
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Check License
+        if: matrix.runner == 'ubuntu'
+        uses: apache/skywalking-eyes@9bd5feb86b5817aa6072b008f9866a2c3bbc8587
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kezhenxu94) [inline .github/workflows/rover.yaml:40]: BTW, we don't need this anymore, the token can be obtained in the action automatically

File: .github/workflows/rover.yaml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Check License
+        if: matrix.runner == 'ubuntu'
```
Comment (kezhenxu94) [inline .github/workflows/rover.yaml:37]: This `if` condition is also redundant, we only run in `ubuntu` runner

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**